src/backend/common/redis.py

```diff
@@ -0,0 +1,24 @@
+import os
+from typing import Optional
+
+import redis
+from pyre_extensions import none_throws
+
+
+class RedisClient:
+    """
+    This will let us share redis clients across use cases in the code
+    """
+
+    client: Optional[redis.Redis] = None
+
+    @classmethod
+    def get(cls) -> Optional[redis.Redis]:
+        redis_url = os.environ.get("REDIS_CACHE_URL")
```

Comment:
Thoughts on putting this in the `Environment` and then pulling it from there? I've been trying to sub out any os.environ` stuff to all live in `Environment`, and then pass the `Environment` variables in via inits so things are less magic https://github.com/the-blue-alliance/the-blue-alliance/pull/3162/files#diff-23ddd72bfb97c613cc2ea8396a5daf356f1381d836b168cd8525909866cbc30bR35

Reply:
:+1: